modules/localStorage.js

```diff
@@ -0,0 +1,35 @@
+function setInputs(bookCollection) {
+  const booksArr = JSON.parse(localStorage.getItem('books'));
+  bookCollection.setBooks(booksArr);
+}
+
+function populateStorage(bookCollection) {
+  localStorage.setItem('books', JSON.stringify(bookCollection.getBooks()));
+  setInputs(bookCollection);
+}
+
+function storageAvailable(type) {
+  let storage;
+  try {
+    storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (e) {
+    return e instanceof DOMException && (
+      // everything except Firefox
+      e.code === 22
+              // Firefox
+              || e.code === 1014
+              // test name field too, because code might not be present
+              // everything except Firefox
+              || e.name === 'QuotaExceededError'
+              // Firefox
+              || e.name === 'NS_ERROR_DOM_QUOTA_REACHED')
+              // acknowledge QuotaExceededError only if there's something already stored
+              && (storage && storage.length !== 0);
+  }
+}
+
```

Comment:
- [ ] Why not make use of arrow syntax for functions as arrow functions are part of ES6 and it is a requirement in this project.